google-cloud-asset/samples/quickstart.rb

```diff
@@ -99,3 +99,53 @@ def create_feed project_id:, feed_id:, pubsub_topic:, asset_names:
   puts "Created feed: #{response.name}"
   # [END asset_quickstart_create_feed]
 end
+
+def search_all_resources scope: "", query: "", asset_types: [], page_size: 0, page_token: "", order_by: ""
+  # [START asset_quickstart_search_all_resources]
+  require "google/cloud/asset/v1"
```

Comment:
Please use `require "google/cloud/asset"` instead, following the other quickstart examples.